router/router.go

```diff
@@ -1133,16 +1133,20 @@ func (*workerT) sendDestinationResponseToConfigBackend(payload json.RawMessage,
 }
 
 func durationBeforeNextAttempt(attempt int) (d time.Duration) {
-	b := backoff.NewExponentialBackOff()
-	b.InitialInterval = minRetryBackoff
-	b.MaxInterval = maxRetryBackoff
-	b.RandomizationFactor = 0
-	b.MaxElapsedTime = 0
-	b.Multiplier = 2
-	b.Reset()
-	for index := 0; index < attempt; index++ {
-		d = b.NextBackOff()
-	}
+	constantMin := float64(minRetryBackoff)
+	/*
+		1 - e^(-0.03125x) ------- 0.03125 based on default values for min(10), max(300) backoff values
+		attempts:
+			1 - 10
+			2 - 18.17
+			3 - 26.84
+			4 - 35.25
+			5 - 43.39
+			6 - 51.27
+	*/
+	exponentialMultiplier := 1 - math.Exp(-float64(attempt)*0.03125)
+	exponentialDuration := float64(maxRetryBackoff) * exponentialMultiplier
+	d = time.Duration(math.Max(constantMin, exponentialDuration))
```

Comment:
Given `minRetryBackoff`, `maxRetryBackoff`, `multiplier: 2` and `randomizationFactor: 0`, shouldn't it look like below?

```
nextBackoff := time.Duration(math.Min(float64(maxRetryBackoff), float64(minRetryBackoff)*math.Exp2(float64(attempt))))
```

Reply:
we need some tests too :)

Reply:
added some